proof/append.go

```diff
@@ -191,6 +232,7 @@ func CreateTransitionProof(prevOut wire.OutPoint,
 			return nil, fmt.Errorf("root asset mismatch")
 		}
 
+		// TODO(jhb): add STXO inclusion proof for root prevIDs
```

Comment:
Marker to be resolved before merge?

Reply:
Oli has the same train of thought as do I. But I want this to be confirmed by @Roasbeef

Reply:
Resolved. See: https://github.com/lightninglabs/taproot-assets/pull/1453#issuecomment-2857973135